css/style.css

```diff
@@ -165,4 +165,29 @@ nav ul li a:hover{
 
 textarea{
     resize: none;
-}
\ No newline at end of file
+}
+
+.footer{
+    text-align: center;
+    font-weight: 600;
+    margin: 4rem 3.3rem 1.5rem;
+}
+
+.footer h4{
+    font-size: 1.7rem;
+    font-weight: 600;
+    padding-bottom: 1rem;
+}
+
+.footer img{
+    align-items: center;
+    width: 2rem;
+    align-items: center;
+    margin: 0.5rem 0 2.5rem 0.6rem;
```

Comment:
Could you consider adding hover effects to the social icons? And I also think naming them `.social_icons` would be a better and explanatory name than trying to style them with the HTML selector.